Background Scripts/Adding bookmark to Favorites tab/Adding bookmark into Favorites tab.js

```diff
@@ -0,0 +1,28 @@
+var memGroups = []; // empty array for getting the group sys'id rin future.

```

Comment:
FYI there is an OOB Dynamic query for My Groups so it's not needed to query the group table and build this array.